### PR TITLE
Update: Microsoft Azure Icon and Color

### PR DESCRIPTION
**Pull request by moodledan**
Microsoft Azure

**Issue:** #5661 
Alexa rank: N/A

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
Source, taken from the Microsoft Azure App Service extension for VS Code repo: https://github.com/microsoft/vscode-azureresourcegroups/blob/0a06362e82170fd7f8dc2496286825b1a69cc42b/resources/azure.svg

I've updated the HEX value for the icon, its the only color that forms part of the sourced icon that isn't a gradient start/stop value, it's also the primary accent/link colour used on the Microsoft Azure portal.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5607,8 +5607,8 @@
         },
         {
             "title": "Microsoft Azure",
-            "hex": "0089D6",
-            "source": "https://developer.microsoft.com"
+            "hex": "0078D4",
+            "source": "https://portal.azure.com"
```
Comment (adamrusted) [inline _data/simple-icons.json:5611]: While I'm reviewing the SVG - could you please update the source URL to the GitHub URL you used for this new version?

**Review by adamrusted — CHANGES_REQUESTED** [review]
Thanks for the updates @moodledan - the icon matches up perfectly, though there are a few [excess points in the paths](https://user-images.githubusercontent.com/1830867/119240625-c01b8300-bb48-11eb-9226-8d2e2ea48c53.png) which could probably be tidied up before optimization for an even smaller file.

**Review by moodledan — COMMENTED** [review]
Excess points removed and increased the floatPrecision value to 3 as per default for the project.

**Review by moodledan — COMMENTED** [review]
Tried to further optimise but then lint check fails so this is the compromise

**Review by adamrusted — APPROVED** [review]
Looks good to me, thank you for your work on this @moodledan!
There are a couple of minor points that _may_ be improved at some point, but as you say they look like they're likely to mess with the geometry, so let's leave them for now. It's worth noting that the word 'icon' is no longer required in the `<title>` tag - so if you could merge develop into your branch, and then remove the word - this'll be good to go!

**Comment by adamrusted** [comment]
Thanks for merging develop in, @moodledan - though it looks like the Azure icon itself has also reverted!
Would you mind resubmitting your edited SVG?

**Comment by moodledan** [comment]
What a n00b, resubmitted now.

**Review by adamrusted — APPROVED** [review]
Spot on, thanks for your work on this @moodledan 🚀

**Pull request merged**